src/localization/de.js

```diff
@@ -19,7 +19,67 @@ export default {
+    'dem Einkommen, der Nummer von Personnen im Haushalt, vielen verschiedenen anderen ' + 
+    'Krieterien, und den Auswirkungen der Programmen selbst haben Auswirkungen ' + 
+    'aufeinander in unerwarteten Weisen.  Wir suchen Methoden, um dieses Problem sich ' + 
+    'mit der Komplexität zu befassen, und die Familien ihre Situationen besser zu ' + 
+    'verstehen und vorherzusehen.',
+    howToUseHeader:  'Wie benutze ich dieses Werkzeug?',
+    howToUse1:  'Gehen Sie Schritt für Schritt, Informationen über eines ' + 
+    'Klienten gegenwärtigen Unterstützungen, Haushalt, Einkommen, und andere wichtige ' + 
+    'Information in das Formular auszufüllen.  Diese Informationen werden benutzt ' + 
+    'werden, um die umgefähre Unterstützung vom Klienten vorherzusehen.  Als Sie ' + 
+    'zum Ende kommen, wechseln Sie den Betrag vom \'Einkommen in Zukunft\', um die ' + 
+    'daraus resultierende Veränderung des Unterstützungsbetrag zu sehen.  Zu dieser ' + 
+    'Zeit können Sie Auswirkungen für die Programmen SNAP und Section 8 Housing ' + 
+    'Voucher sehen.  Bitte bemerken Sie, daß die Vorhersagen sich vielleicht nicht ' + 
+    'direkt zu dem gegenwärtigen Unterstützungsbetrag eines Klienten passen.  Das ' + 
+    'Ziel der Anwendung ist der Betrag von dem Unterstützungswechsel, das sich ' + 
+    'ereignet, als es Einkommenwechsel gibt.',
+    howToUseRefreshNote:  'Bitte bemerken Sie, daß diese Anwendung die Benutzerdaten nicht ' + 
+    'behaltet.  Wenn Sie die Webseite wieder aufnehmen, wird Ihre Daten verloren ' + 
+    'werden.  Jedes Mal, wenn Sie die Anwendung verwenden, gibt die Anwendung Ihnen ' + 
+    'ein neues, leeres Formular.',
+    whoMadeThisHeader:  'Wer entwickelt diese Anwendung?',
+    whoMadeThis1A:  'Diese Anwendung ist ein Teil eines Projektes, das möglich mit ' + 
+    'dem Geben einen Open Door Grant (Zuschuß) von der Stiftung Boston Foundation an ' + 
+    'das ',
+    whoMadeThis1B:  ' an der University of Massachusetts Boston ist.  Das Center for ' + 
+    'Social Policy arbeitet mit den Organisationen ',
+    whoMadeThis1C:  ' und ',
+    whoMadeThis1D:  '.  Das Center for Social Policy ist der Hauptpartner für das ',
+    whoMadeThis1E:  '.',
+    whoMadeThis2A:  'Die Freiwilligen von ',
+    whoMadeThis2B:  ' erhalten die Codebasis auf ',
+    whoMadeThis2C:  ' aufrecht.  Für weitere Auskunft treten Sie bitte in Verbindung mit ',
+    whoMadeThis2D:  '.',
+    whoMadeThis3:  'Wir möchten den folgenden Freiwilligen von Code for Boston ' + 
+    'besonders danken: Annie LaCourt, Isaac Chansky, Michelle Bernstein, Alec Danaher, ' + 
+    'Sasha Maryl, Drew Love, Liani Lye, Andrew Cunningham, Liam Morley, Nick Francisci, ' + 
+    'Stephen Chin, Shameek Poddar, Will McIntosh, Andrew Seeder, Ben Lewis, Don Blair, ' + 
+    'Ethan Strominger, Nick Lee, Jonathan Marcus, Emily Wasserman, Ethan Blackwood, ' + 
+    'und Valerie Kenyon.',
```

Comment:
Actually, I think there'd have to be `whoMadeThis3`, `whoMadeThisNames`, and `whoMadeThis4`, so that english could fill in the names. Though, honestly, we may want to think about these property names to see if we can make them more descriptive.

Does this match the English version of this object somewhere? I haven't seen that. If that doesn't exist, it may be easier to make the English one first, and then base other languages off of that.

Reply:
Yeah, maybe...I think it might be better to keep them on the about page itself though, since they'll never be translated, and it probably makes more sense to newcomers to go there to add their name.

Reply:
Just as an idea, I've broken up whoMadeThis3 into 5 new properties:
whoMadeThis3Intro
whoMadeThis3Names
whoMadeThis3And
whoMadeThis3FinalName
whoMadeThis3Period
But I also wonder if doing the name list in English, without trying to translate "and," is okay.  Or maybe if we use "+" or "&" to simplify the names list into one string.

Reply:
Hmm, worth thinking about. Would it be considered less professional?

Reply:
Also, would we have to use it throughout the page to be consistent?